spec/tbd_tests_spec.rb

```diff
@@ -1424,6 +1429,159 @@
       end
     end
 
-    expect(nb).to eq(10) # Twice 5x: each edge is either object or subject
+    expect(nb).to eq(10) # Twice 5x: each edge is once object, once subject
+
+    dads = {}
+
+    subs.keys.each do |id|
+      kid = model.getSubSurfaceByName(id)
+      expect(kid.empty?).to be(false)
+      kid = kid.get
+      dad = kid.surface
+      expect(dad.empty?).to be(false)
+      dad = dad.get
+      nom = dad.nameString
+      expect(surfaces.key?(nom)).to be(true)
+      loss = surfaces[nom][:heatloss]
+      dads[nom] = loss
+
+      case nom
+      when "Entryway  Wall 4"     then expect(loss).to be_within(TOL).of(2.705)
+      when "Entryway  Wall 5"     then expect(loss).to be_within(TOL).of(4.820)
+      when "Entryway  Wall 6"     then expect(loss).to be_within(TOL).of(2.008)
+      when "Smalloffice 1 Wall 1" then expect(loss).to be_within(TOL).of(5.938)
+      when "Smalloffice 1 Wall 2" then expect(loss).to be_within(TOL).of(3.838)
+      when "Smalloffice 1 Wall 6" then expect(loss).to be_within(TOL).of(3.709)
+      when "Utility1 Wall 1"      then expect(loss).to be_within(TOL).of(5.472)
+      when "Utility1 Wall 5"      then expect(loss).to be_within(TOL).of(5.440)
```

Comment:
Heat loss from major thermal bridging (in W/K), _WITHOUT_ having TBD reset any window **jambs** as (mild) **transitions** as a result of a user-set proximity tolerance of e.g. 100mm.